test/metabase/query_processor/middleware/cache_test.clj

```diff
@@ -248,6 +248,7 @@
       (mt/wait-for-result save-chan)
       (is (= :cached (run-query))))))
 
+#_
```

Comment:
probably worth writing a comment saying why this test is commented out next time, otherwise someone will look at it and be confused and the only way they could figure out why would be to go hunt down this Git commit message/PR